openml/openml_simple.py

```diff
@@ -0,0 +1,119 @@
+"""
+====================================================
+Hyperparameter Optimization Benchmark with OpenML
+====================================================
+
+In this tutorial, we walk through how to conduct hyperparameter optimization experiments using
+OpenML and OptunaHub.
+"""
+
+############################################################################
+# Please make sure to install the dependencies with:
+# ``pip install -r requirements.txt``
+# Then we import all the necessary modules.
+
+# License: BSD 3-Clause
```

Comment:
Since the Optuna organization uses the MIT license, it would be preferable to adopt the same license for this project as well.

Reply:
Sounds good, done :)